### PR TITLE
[KRC-73] Style the styleguide

### PR DESCRIPTION
**Pull request by blurbyte**
[KRC-73](https://digitalrig.atlassian.net/browse/KRC-73) 

Screenshot:
Screenshot 2019-04-20 at 22 21 30

- [x] Added styleguide related npm scripts (and updated deploy one)
- [x] Created custom logo and components list
- [x] Updated code editor look
- [x] Updated export statement in most _index.js_ files to shorter form

### CODE REVIEW

**Review by mgolawski — APPROVED** [review]
👯

File: src/components/ShippingDetails/__tests__/__snapshots__/ShippingDetails.test.js.snap [hunk 1]
```diff
@@ -182,7 +182,7 @@ Array [
         <span
           className="c2"
         >
-          Fri, Apr 19 - Mon, Apr 22
+          Tue, Apr 23 - Fri, Apr 26
```
Comment (lucianojss) [inline src/components/ShippingDetails/__tests__/__snapshots__/ShippingDetails.test.js.snap]: red flag! This is a bad test 🐛, `ShippingDetails` component there is an external function `formatDate` that is not being mocked. It is not related with this PR I know.

Comment (blurbyte) [reply src/components/ShippingDetails/__tests__/__snapshots__/ShippingDetails.test.js.snap]: I fixed in in previous PR :)

Comment (lucianojss) [reply src/components/ShippingDetails/__tests__/__snapshots__/ShippingDetails.test.js.snap]: So maybe is better to remove this file from the changes 👍

**Review by lucianojss — CHANGES_REQUESTED** [review]

**Review by lucianojss — APPROVED** [review]

**Pull request merged**